### PR TITLE
[30]: Enable post to FB

### PR DESCRIPTION
**Pull request by JacobPernell**
# Description
Issue: https://github.com/mskalandunas/Robin/issues/30
Description: Using Postman to send a POST request to the /content route, a function has been added to handle sending an HTTP POST request to the Facebook Graph API to post on a Facebook Page.

Right now the `message` parameter as part of the request body is hard-coded (which might result in an error if you try to send multiple requests with the same `message` value), but that will be changed in the future once we start handling information sent from the client side.

# Screenshots/GIFS

# Reviewers
@mskalandunas

### CODE REVIEW

**Comment by JacobPernell** [comment]
Alright right now the function is named `handleFacebook`, but in the future I think I'll need to rewrite it to be more generic and handle any kind of request from multiple social platforms.

File: server/server.go [hunk 1]
```diff
@@ -11,14 +16,40 @@ type ContentResponseMeta struct {
...
-	Body string `json:"body"`
+	ContentBody string `json:"content_body"`
 }
 
 type ContentResponse struct {
 	Meta ContentResponseMeta `json:"meta"`
 	Data []Content `json:"data"`
 }
 
+func requestFacebook(c echo.Context) error {
```
Comment (mskalandunas) [inline server/server.go:27]: Unless the function signature provides a parameter for describing the type of request and the type of endpoint, i think this function name needs to be much more specific to the type of resource being requested.

File: server/server.go [hunk 2]
```diff
@@ -11,14 +16,40 @@ type ContentResponseMeta struct {
...
+  postURL := fmt.Sprintf("https://graph.facebook.com/%s/feed", FB_PAGE_ID)
+
+  postBody, _ := json.Marshal(map[string]string{
+    "message":  "Hello from Go!",
+    "access_token": FB_PAGE_ACCESS_TOKEN,
+  })
+  reqBody := bytes.NewBuffer(postBody)
+
+  resp, err := http.Post(postURL, "application/json", reqBody)
```
Comment (mskalandunas) [inline server/server.go:36]: is `resp` a named variable the `http.POST` method returns?  If not, you should feel no need to abbreviate.  Abbreviations lead to less readable code.  Descriptive variables will always be for the best.

**Comment by mskalandunas** [comment]
@JacobPernell it looks like you have conflicts because maybe you didn't rebase your branch against `main` when you started working on this part.

**Comment by JacobPernell** [comment]
I'm not sure if I solved this merge conflict correctly, but it is unblocked now. @mskalandunas

**Review by mskalandunas — APPROVED** [review]

**Pull request merged**